src/main/java/com/google/api/codegen/config/DiscoveryField.java

```diff
@@ -195,12 +195,13 @@ public Cardinality getCardinality() {
 
   @Override
   public boolean isEnum() {
-    return schema.isEnum();
+    // TODO(andrealin): implement.
+    return false;
```

Comment:
what happened to `schema.isEnum()`?

Reply:
I think the continued enablement of isEnum() might have thrown an exception somewhere further down the generator that didn't know how to handle enums. I can do it later, outside of this unit_test PR.